df_py/challenge/test/test_judge.py

```diff
@@ -16,18 +18,20 @@ def test_get_txs():
 
     with patch("df_py.util.graphutil.submit_query") as mock:
         mock.return_value = {
-            "nftTransferHistories": [
-                {
-                    "timestamp": six_days_ago.timestamp(),
-                    "nft": {"id": "0xnft1"},
-                    "oldOwner": {"id": "0xfrom1"},
-                },
-                {
-                    "timestamp": one_day_ago.timestamp(),
-                    "nft": {"id": "0xnft2"},
-                    "oldOwner": {"id": "0xfrom2"},
-                },
-            ]
+            "data": {
+                "nftTransferHistories": [
+                    {
+                        "timestamp": six_days_ago.timestamp(),
+                        "nft": {"id": "0xnft1"},
+                        "oldOwner": {"id": "0xfrom1"},
+                    },
+                    {
+                        "timestamp": one_day_ago.timestamp(),
+                        "nft": {"id": "0xnft2"},
+                        "oldOwner": {"id": "0xfrom2"},
```

Comment:
I haven't looked at the rest of judge-related logic, and this is extremely minor/not the changelist. but, is `oldOwner` supposed to be `originalOwner`?